CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
- # Copyright (c) 2023 General Motors GTO LLC
+ # Copyright (c) 2024 General Motors GTO LLC
```

Comment:
Why are copyright dates being updated in files that have not otherwise changed? By my count, _at least_ 34 of the 62 files changed in this PR _only_ update the copyright year. Unless there is a good reason to do this, all 34 of these files should all be reverted.